.gitignore

```diff
@@ -48,6 +48,7 @@ lib/src/localization/gen/*
 #.vscode/
 
 # Flutter/Dart/Pub related
+.fvm/
```

Comment:
Do you think it would be tidier to keep these in the Custom section above, so
all changes from the default are together?

Reply:
The .fvm folder is not related to the overall PR, it's just handy to have it there to ignore the flutter version manager files.

dist/ could be put somewhere else if you think it's worthwhile to do so.